src/main/cpp/VLFeat.cxx

```diff
@@ -19,13 +19,14 @@
 
 #include <iostream>
 #include <fstream>
-   
```

Comment:
While I appreciate code cleanliness, it looks like your editor is automatically removing trailing whitespaces. This makes these diffs harder to read - would be great if you can turn that off for future PRs (or we can do this once on the whole codebase and make it a style guideline...)